sys/net/gnrc/netif/init_devs/auto_init_nrf802154.c

```diff
@@ -46,13 +51,24 @@ void auto_init_nrf802154(void)
     LOG_DEBUG("[auto_init_netif] initializing nrf802154\n");
 
     netdev_register(&nrf802154_netdev.dev.netdev, NETDEV_NRF802154, 0);
-    netdev_ieee802154_submac_init(&nrf802154_netdev);
-    nrf802154_hal_setup(&nrf802154_netdev.submac.dev);
 
     nrf802154_init();
-    gnrc_netif_ieee802154_create(&_netif, _stack,
+#if IS_USED(MODULE_OPENDSME)
```

Comment:
> Want to add this for auto_init_socket_zep.c anyway? Compiling on native can be beneficial for several things, even when it crashes at run time due to the known floating point issue.

The problem is, `socket_zep` does not add a mechanism to estimate the RX start timestamp, which is used for the timeslot calculation. So, it would not work, even if the `native` floating point issue is fixed.

> What about cc2538?

I haven't tested it again with that one. I can give it a try

Reply:
It didn't work out of the box, I suspect it's something related to timings.
I propose to blacklist it for know and try to merge it for the nrf52 radios. I can then make it work for cc2538 in a follow up